docs/blockchain-users/FAQ.md

```diff
@@ -33,3 +37,60 @@ The [Discourse forum](https://quilibrium.discourse.group/) is intended to be the
 Discord especially, as the project started from a discord alternative. 
+
+### How fast is Quilibrium?
+
+The Quilibrium network has been tested to handle up to 100 million messages per second (MPS), demonstrating its capability to support highly demanding applications such as social networks and financial services for processing payments or trades.
+
+While MPS is not exactly the same as transactions per second (TPS), they are closely related. The actual TPS depends on the number of nodes available to handle the load. Initial estimates, based on previous node count statistics, suggest a realistic upper bound of 1.5-2.5 million TPS.
+
+### How does Quilibrium balance privacy with preventing criminal activities?
+
+1. Using a cryptographic accumulator bundle in each coin to assert funds are not from illicit/sanctioned addresses.
+2. Implementing a suboptimal default interaction model for core shards, discouraging abuse.
+3. Planning to incorporate specialized intrinsics for content moderation, especially for image hosting.
+4. Allowing node operators to maintain blacklists of problematic core shards, based on revealed decryption keys or compliance entity certifications.
+
+This approach aims to preserve user privacy while providing compliance measures and discouraging criminal exploitation of the network.
+
+### How does Quilibrium prevent centralization?
+
+1. Not requiring expensive hardware for consensus - even a Raspberry Pi can perform necessary calculations.
+2. Allowing various types of hardware to participate in mining due to flexible application complexity.
+3. Enforcing minimum replication thresholds for core shards, inducing halts if they fall below the threshold.
+4. Designing its architecture to alleviate centralization pressures in areas like wallets and indexers.
+
+### What are people building on Q right now?
+
+Not everything is ready to share publicly just yet but several public community projects include collectables/NFTs, node managers, Howler (flagship demo app, Discord alternative) and decentralized exchanges.
+
+### What types of applications can be developed on Quilibrium?
+
+Quilibrium will be able to host any type any application, but the first generation of applications will probably fall in these categories:
+1. Social Media: image hosting, long-form content, social graphs
+2. Financial: tokenized assets, DeFi
+3. Common Web Dev: SPAs, file hosting
+
+### What software knowledge will be required for applications to be developed on Quilibrium?
+
+For network native development, QCL, as a subset of golang, is the primary language. Building new intrinsics can be done in many different languages, ideally golang, rust, or C++.\
+Local simulator and SDKs are coming up shortly.
+
+### What are the core technologies used in Quilibrium?
+
+Here's a very short list of Quilibrium's core technologies with brief descriptions:
+
+### What are the core technologies used in Quilibrium?
```

Comment:
I love this section! Could you please also link to the relevant pages in the Learn section?